adafruit_hid/us_keyboard_layout.py

```diff
@@ -0,0 +1,218 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2017 Scott Shawcroft for Adafruit Industries
```

Comment:
I wasn't sure how Ada handles copyrights from contributed code from non-employees. I can just hold the copyright or assign it to Adafruit. Do you have management guidance on this?